Sources/Widgets/Widgets3D/ResliceCursorWidget/example/index.js

```diff
@@ -122,8 +165,28 @@ for (let i = 0; i < 3; i++) {
   obj.resliceMapper.setInputConnection(obj.reslice.getOutputPort());
   obj.resliceActor = vtkImageSlice.newInstance();
   obj.resliceActor.setMapper(obj.resliceMapper);
+  obj.sphereActors = [];
+  obj.sphereSources = [];
 
-  viewAttributes.push(obj);
+  // Create sphere for each 2D views which will be displayed in 3D
+  // Define origin, point1 and point2 of the plane used to reslice the volume
+  for (let j = 0; j < 3; j++) {
+    const sphere = vtkSphereSource.newInstance();
+    sphere.setRadius(10);
+    const mapper = vtkMapper.newInstance();
+    mapper.setInputConnection(sphere.getOutputPort());
+    const actor = vtkActor.newInstance();
+    actor.setMapper(mapper);
+    actor.getProperty().setColor(...viewColors[i]);
+    obj.sphereActors.push(actor);
+    obj.sphereSources.push(sphere);
+  }
```

Comment:
you might want to create a "Debug" variable and show those spheres only when that Debug variable is `true`. It should be `false` by default.